docs/docs/cli.md

```diff
@@ -1,91 +1,204 @@
+
+#### `--output-class-names`
+
+Setting this option to `True` yields the most concise output `zamba` is capable of. The highest species probability in a video is taken to be the _only_ species in that video, and the output returned is simply the video name and the name of the species with the highest class probability, or `blank` if the most likely classification is no animal. See the [Quickstart](index.md) for example usage.
+
+#### `--weight_download_region TEXT` <!-- TODO: make sure CLI arg. doesn't show up now but it should be one><!-->
 
 Because `zamba` needs to download pretrained weights for the neural network architecture, we make these weights available in different regions. 'us' is the default, but if you are not in the US you should use either `eu` for the European Union or `asia` for Asia Pacific to make sure that these download as quickly as possible for you.
 
-#### --verbose BOOLEAN
+#### `--yes`
+
+By default, the command line interface will print the specifications you have entered and ask for confirmation before starting inference. Specifying `--yes` or `-y` skips this step and kicks off prediction without confirmation. 
+
+## `zamba train`
+
+```
+$ zamba train --help
+Usage: zamba train [OPTIONS]
+
+  Train a model using the provided data, labels, and model name.
+
+  If an argument is specified in both the command line and in a yaml file,
+  the command line input will take precedence.
+
+Options:
+  --data-dir PATH                 Path to folder containing videos.
+  --labels PATH                   Path to csv containing video labels.
+  --model [time_distributed|slowfast]
+                                  Model class to train.  [default:
+                                  time_distributed]
+
+  --config PATH                   Specify options using yaml configuration
+                                  file instead of through command line
+                                  options.
+
+  --batch-size INTEGER            Batch size to use for training.
+  --gpus INTEGER                  Number of GPUs to use for training. If not
+                                  specifiied, will use all GPUs found on
+                                  machine.
+
+  --dry-run / --no-dry-run        Runs one batch of train and validation to
+                                  check for bugs.
+
+  -y, --yes                       Skip confirmation of configuration and
+                                  proceed right to training.  [default: False]
+
+  --help                          Show this message and exit.
+```
+
+All of the above besides `model` and `yes` can also be specified in a yaml file.<!--TODO: is that list accurate?><!--> Let's go through the flags one by one.
+ 
```

Comment:
Is the above (`model` and `yes`) an accurate list of the params that can't be specified in the yaml for train?

Reply:
Yep, should be accurate.